django/core/files/storage/__init__.py

```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
```

Comment:
Adding the `storage` module should be made in advance in a separate PR. We don't want to mix new features with cleanups.

Reply:
Yes, good point. Should I create a ticket about this topic myself?

Reply:
There is no need for a separate ticket, you can open PR titled `Refs #34110 -- ...`.

Reply:
Ok thanks 👍